tldw_Server_API/app/services/admin_webhooks_service.py

```
@@ -0,0 +1,442 @@
+        webhooks, _ = await self.list_webhooks(active_only=True, limit=1000)
+        matching = [
+            wh for wh in webhooks
+            if "*" in wh.event_types or event_type in wh.event_types
+        ]
+        if not matching:
+            return 0
+
+        async def _deliver_one(wh: WebhookRecord) -> bool:
+            try:
+                await self.deliver(wh, event_type, payload)
+                return True
```

Ruff (0.15.7)

[warning] Consider moving this statement to an `else` block

(TRY300)